R/tbl_survfit_quantiles.R

```diff
@@ -0,0 +1,245 @@
+#' Survival Quantiles
+#'
+#' Create a gtsummary table with Kaplan-Meier estimated survival quantiles.
```

Comment:
I would add a comment to the very nice chapter ARD-first

Reply:
What is that? 😆

Reply:
Add note saying the `tbl_*()` function is a shortcut, and if you need to customize see the Details below on ARD-first

Reply:
Added! ✔️